ogr/services/github/pull_request.py

```diff
@@ -114,8 +114,11 @@ def merge_commit_sha(self) -> str:
         return self._raw_pr.merge_commit_sha
 
     @property
-    def mergeable(self) -> bool:
-        return self._raw_pr.mergeable
+    def merge_status(self) -> str:
+        if self._raw_pr.mergeable:
+            return "can_be_merged"
```

Comment:
I thought you meant an enumeration like [this](https://github.com/packit/ogr/blob/79d3a39a239d2fbd5959645dfc3b481785547adb/ogr/abstract.py#L591-L597) for commit statuses.

Reply:
agreed, an enum here would be much better